synapse/config/server.py

```diff
@@ -81,6 +81,7 @@ def read_config(self, config):
         # Options to disable HS
         self.hs_disabled = config.get("hs_disabled", False)
         self.hs_disabled_message = config.get("hs_disabled_message", "")
+        self.hs_disabled_limit_type = config.get("hs_disabled_limit_type", "")
```

Comment:
We should add some documentation about what these options do. Traditionally, we usually do this buy including them in the default config below (commented out where appropriate)